documentation/src/main/asciidoc/topics/proc_using_java_serialization.adoc

```diff
@@ -1,29 +1,25 @@
-[id='java_serialization']
-= Using Java Serialization
-You can use Java serialization with {brandname} to marshall your objects, but
-only if your Java objects implement the Java `Serializable` interface.
+[id='using-java-serialization_{context}']
+= Using Java serialization
+You can use Java serialization with {brandname} to marshall your objects, but only if your Java objects implement the Java `Serializable` interface.
```

Comment:
not sure if relevant, but it should work with `java.io.Externalizable` too.

Reply:
maybe that's TMI but I've reworded this to make it more concise